src/Base/Placement.cpp

```diff
@@ -209,3 +210,23 @@ Placement Placement::sclerp(const Placement& p0, const Placement& p1, double t,
     Placement trf = p0.inverse() * p1;
     return p0 * trf.pow(t, shorten);
 }
+
+void Placement::print(std::string name)
+{
+
+    Base::Vector3d pos = getPosition();
+    Base::Vector3d axis;
+    double angle;
+    Base::Rotation rot = getRotation();
+    rot.getRawValue(axis, angle);
+    Base::Console().Warning(

```

Comment:
Why is it a warning? Isn’t it a neutral information?

Reply:
It's just for development. In this case you want to see the message. I personally use warning for development because log are too verbose.